extensions/custom-header/src/Extension.jsx

```diff
@@ -1,11 +1,79 @@
-import {Image} from '@shopify/ui-extensions-react/checkout';
+import {
+  Divider,
+  InlineLayout,
+  Link,
+  Text,
+  View,
+  useBuyerJourneySteps,
+  useBuyerJourneyActiveStep,
+  useShop,
+  useTranslate,
+} from '@shopify/ui-extensions-react/checkout';
 
-// [START custom-header.render]
 export default function Extension() {
+  // [START custom-header.buyer-journey]
+  const steps = useBuyerJourneySteps();
+  const activeStep = useBuyerJourneyActiveStep();
+  const {storefrontUrl} = useShop();
+  const translate = useTranslate();
+
+  const assembledSteps = [
+    {label: translate('cart'), handle: 'cart', to: `${storefrontUrl}cart`},
```

Comment:
Something mentioned on a previous tutorial was that we should avoid setting the expectation that the storefrontUrl will always have a trailing slash. Using a URL constructor will assure it's formatted correctly.

```suggestion
    {label: translate('cart'), handle: 'cart', to: new URL('/cart', storefrontUrl).href},
```

Reply:
great call ty